dev/io.openliberty.java.internal_fat/publish/servers/java17-cdi-server/server.xml

```diff
@@ -0,0 +1,24 @@
+<!--
+    Copyright (c) 2021 IBM Corporation and others.
```

Comment:
Please update the copyright date since this is a new file.

Reply:
Done